packages/millicast-sdk/src/Publish.js

```diff
@@ -124,4 +124,9 @@ export default class Publish extends BaseWebRTC {
     this.setReconnect()
     logger.info('Broadcasting to streamName: ', this.streamName)
   }
+
+  reconnect () {
+    this.options.mediaStream = (this.webRTCPeer?.peer) ? this.webRTCPeer.peer.getSenders().map((sender) => sender.track) : this.options.mediaStream
```

Comment:
It would be better to mainting RTCPeerConnection access inside our PeerConnection class. Consider a new method in PeerConnection called `getActiveTracks()`